bots/github/__init__.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, base=GITHUB_BASE, cacher=None):
 
         # The cache directory is $TEST_DATA/github bots/github
         if not cacher:
-            directory = os.path.join(os.environ.get("TEST_DATA", BOTS), "github")
+            directory = os.path.join(os.environ.get("TEST_DATA", BOTS), ".github-cache")
```

Comment:
For the runners TEST_DATA is ```/cache``` and ```/cache/github``` is a well known thing. So I would suggest changing the BOTS fallback for TEST_DATA here instead.